gatsby/content/blog/2022/07/2022-08-03-synapse-1.64.mdx

```diff
@@ -0,0 +1,98 @@
+---
+date: '2022-08-03'
+title: Synapse 1.64 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+It's that time again: there's a new Synapse release, fresh out of the oven!
+Let's take a look at what's inside [Synapse
+1.64](https://github.com/matrix-org/synapse/releases/tag/v1.64.0).
+
+## Delegating email verification is now deprecated
+
+[Synapse 1.4.0](https://github.com/matrix-org/synapse/releases/tag/v1.4.0)
+introduced a configuration option (`account_threepid_delegates.email`) to allow
+homeservers to delegate validating the ownership of email addresses to an
+external identity server. This validation is used by Synapse when associating
+email addresses to a Matrix account, or before performing a password reset.
+
+As of Synapse 1.64, this option is deprecated. This is because this option
+relies on old API endpoints that have since been removed from the Matrix
+specification. Server administrators who still need the ability to validate the
+ownership of email addresses must configure an SMTP server in their Synapse
+configuration file. See the [configuration
+guide](https://matrix-org.github.io/synapse/v1.64/usage/configuration/config_documentation.html#email)
+for more information.
```

Comment:
How about something like

```suggestion
specification.

Synapse can do this validation internally provided it is configured with
details of an SMTP server. Administrators currently relying on
`account_threepid_delegates.email` should therefore ensure that an
SMTP server is correctly configured, and remove the
`account_threepid_delegates.email` option. See the [configuration
guide](https://matrix-org.github.io/synapse/v1.64/usage/configuration/config_documentation.html#email)
for more information.
```

... which I think is a bit of a clearer call to action?